include/robot_state_publisher/robot_state_publisher.hpp

```diff
@@ -34,72 +34,85 @@
 
 /* Author: Wim Meeussen */
 
-#ifndef ROBOT_STATE_PUBLISHER__ROBOT_STATE_PUBLISHER_H_
-#define ROBOT_STATE_PUBLISHER__ROBOT_STATE_PUBLISHER_H_
-
+#ifndef ROBOT_STATE_PUBLISHER__ROBOT_STATE_PUBLISHER_HPP_
+#define ROBOT_STATE_PUBLISHER__ROBOT_STATE_PUBLISHER_HPP_
+
+#include <builtin_interfaces/msg/time.hpp>
+#include <kdl/tree.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <sensor_msgs/msg/joint_state.hpp>
+#include <std_msgs/msg/string.hpp>
+#include <tf2_ros/static_transform_broadcaster.h>
+#include <tf2_ros/transform_broadcaster.h>
+#include <urdf/model.h>
+
+#include <chrono>
 #include <map>
+#include <memory>
 #include <string>
+#include <vector>
 
-#include "kdl/frames.hpp"
-#include "kdl/segment.hpp"
-#include "kdl/tree.hpp"
-
-#include "tf2_ros/static_transform_broadcaster.h"
-#include "tf2_ros/transform_broadcaster.h"
-#include "urdf/model.h"
-
-#include "std_msgs/msg/string.hpp"
-
-#include "rclcpp/rclcpp.hpp"
+typedef std::map<std::string, urdf::JointMimicSharedPtr> MimicMap;
```

Comment:
I generally prefer `using` to `typedef` in modern c++, but I don't think that it makes much of a difference here.

Reply:
This was a leftover from the old code.  But yeah, I'll change to `using`.